oneanddone/tasks/templates/tasks/emails/feedback_email.txt

```diff
@@ -4,4 +4,4 @@ The task: {{ task_name }} ({{ task_link }}) was {{ task_state }}.
 
 Feedback:
 
-{{ feedback }}
+{{ feedback }}({{ feedback_link }})
```

Comment:
The link ends up appearing immediately after the feedback text, which can be a bit confusing. I suggest adding a newline between the `{{ feedback }}` and `({{ feedback_link }})`.

Here is an example email with the current code:
```
---------- MESSAGE FOLLOWS ----------
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: Feedback on Try out the latest Firefox  from One and Done
From: oneanddone@mozilla.com
To: bsilverberg@mozilla.com
Date: Mon, 19 Jan 2015 21:33:07 -0000
Message-ID: <20150119213307.16734.2182@bsilverberg-09386.local>
X-Peer: 127.0.0.1

Here is your One and Done task feedback from Bob Silverberg (bsilverberg@mozilla.com):

The task: Try out the latest Firefox  (http://127.0.0.1:8000/tasks/16/) was Finished.

Feedback:

Here is some feedback.(http://127.0.0.1:8000/admin/tasks/feedback/270)
------------ END MESSAGE ------------
```